core/shared/src/main/scala/zio/ZIO.scala

```diff
@@ -2049,6 +2049,16 @@ private[zio] trait ZIOFunctions extends Serializable {
       acc.flatMap(f(_, el))
     }
 
+  /**
+   * Folds an `Iterable[A]` using an effectful function `f`, working sequentially.
```

Comment:
Let's clarify that this does it from right to left and `foldLeft` does it from left to right.

Reply:
Maybe just "Folds an `Iterable[A]` using an effectual funciton `f`, working sequentially from right to left" and similarly above?

Reply:
Had never heard of the word effectual before. TIL